Update windows dev readme

omgoshjosh opened the pull request:
raise_for_status was just bugging me but i can take it out if it's there for a reason

omgoshjosh committed:
update readme and requirements.txt for non-docker windows dev

dtracers commented on README.md:
```diff
@@ -2,10 +2,12 @@
 ## Setup
 - Install Python 3.6/pip
-- `pip3 install -r requirements.txt`
+- Create a virtual environment `py -m venv venv && source venv/Scripts/activate`, `py` should be however you run python 3
```
We should probably suggest multiple ways to install python and not assume a single workflow.

For example some may want to use pycharm, not use any virtual env or use what is suggested here.

dtracers commented on backend/blueprints/steam.py:
```diff
@@ -67,7 +67,7 @@ def steam_id_to_profile(steam_id):
         steamKey=STEAM_API_KEY, steamID=steam_id)
     resp = requests.get(profile_url)
-    resp.raise_for_status()
+    # resp.raise_for_status()
```
why is this commented out?

omgoshjosh replied:
i dunno, it just seemed like an unnecessary stack trace. like can't we handle this exception without raising? same for the comment below. it was selfishly just for dev purposes, but i do think this could be more gracefully handled. i just don't have enough context yet to propose a good solution. hence just commenting it out.

dtracers replied:
It does not throw on production only local dev.  We could probably wrap it in a is local check

dtracers commented on backend/blueprints/steam.py:
```diff
@@ -120,7 +120,7 @@ def vanity_to_steam_id(vanity):
         steamKey=STEAM_API_KEY, steamID=vanity)
     resp = requests.get(steam_url)
-    resp.raise_for_status()
+    # resp.raise_for_status()
```
Why is this commented out?

omgoshjosh committed:
update requirements.txt and README

for happy/golden path setup and a clean/fresh install in an effor to be
more beginner friendly.
